packages/headless/src/api/generated-answer/generated-answer-event-payload.ts

```diff
@@ -0,0 +1,29 @@
+export type GeneratedAnswerStreamFinishReason = 'COMPLETED' | 'ERROR';
+
+export type GeneratedAnswerPayloadType =
+  | 'genqa.messageType'
+  | 'genqa.citationsType';
+
+export interface GeneratedAnswerCitation {
+  id: string;
+  title: string;
+  uri: string;
+  permanentid: string;
+  clickUri?: string;
+}
+
+export interface GeneratedAnswerMessagePayload {
+  textDelta: string;
```

Comment:
Is this always text or can it be HTML?

If it can be HTML, then I would replace `text` with `html`.

Reply:
you mean you would call it `htmlDelta` instead? This is defined by the schema returned by the API so the ML team and I settled on textDelta a few weeks ago. So far it is text. Could it potentially be rich text down the road with some markup tags? possibly. But we can't really change this in the backend at this point in time.

Reply:
alright then, just gotta make sure that the API never returns HTML in this.

Reply:
If there is html it will be stringified for sure.